e2e/helpers/helper.ts

```diff
@@ -13,9 +13,7 @@ export function waitForPageLoaded() {
 }
 
 export function waitForSpinner() {
-  // return browser.wait(EC.visibilityOf(CommonChartPage.spinner), 3000, 'spinner to be visible').then(() => {
-  return browser.wait(EC.stalenessOf(CommonChartPage.spinner), MAX_TIMEOUT, 'stalenessOf of spinner');
-  // });
+  return browser.wait(EC.invisibilityOf(CommonChartPage.spinner), MAX_TIMEOUT, 'invisibility of spinner');
```

Comment:
Third parameter - it's an error message, which will be thrown in case of failure. Try to make it more descriptive. Example from oficial docs: browser.wait(started, 5 * 1000, 'Server should start within 5 seconds');